### ISSUE TITLE
Exception on log_dataframe() after RAPIDS change

### ISSUE THREAD

**Issue by bernease** [whylabs/whylogs#95]
Calling `log_dataframe()` on either Session or Logger object is producing the following error. This appears to be because we are passing data to the internal `DatasetProfile.track()` incorrectly or `track()` is mishandling the string object.

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-8d9b5da2f1f0> in <module>
      1 sess = whylogs.get_or_create_session()
----> 2 profile = sess.log_dataframe(df)

~/miniconda3/envs/wlprod/lib/python3.8/site-packages/whylogs/app/session.py in log_dataframe(self, df, dataset_name, dataset_timestamp, session_timestamp, tags, metadata)
    151         )
    152 
--> 153         ylog.log_dataframe(df)
    154 
    155         return ylog.close()

~/miniconda3/envs/wlprod/lib/python3.8/site-packages/whylogs/app/logger.py in log_dataframe(self, df)
    148         if not self._active:
    149             return
--> 150         self._profile.track_dataframe(df)
    151 
    152     def is_active(self) -> bool:

~/miniconda3/envs/wlprod/lib/python3.8/site-packages/whylogs/core/datasetprofile.py in track_dataframe(self, df)
    228             x = df[col].values
    229             for xi in x:
--> 230                 self.track(col_str, xi)
    231 
    232     def to_properties(self):

~/miniconda3/envs/wlprod/lib/python3.8/site-packages/whylogs/core/datasetprofile.py in track(self, columns, data)
    182             self.track_datum(columns, data)
    183         else:
--> 184             for column_name, data in list(columns.items()):
    185                 self.track_datum(column_name, data)
    186 

AttributeError: 'str' object has no attribute 'items'
```

**Comment by andyndang** [comment]
I don't know this got introduced but it's probably due to me publishing from my box.

The code looks correct here: https://github.com/whylabs/whylogs-python/blob/bed7a57c1db706fc41e651be4cca81168eac4c66/src/whylogs/core/datasetprofile.py#L184

**Comment by andyndang** [comment]
Should be fixed in 0.1.8
